concrete/controllers/frontend/heartbeat.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Concrete\Controller\Frontend;
+
+use Concrete\Core\Controller\Controller;
+use Concrete\Core\Http\ResponseFactoryInterface;
+use Concrete\Core\Session\SessionValidator;
+
+class Heartbeat extends Controller
+{
+    public function view()
+    {
+        $sessionValidator = $this->app->make(SessionValidator::class);
+        if ($sessionValidator->hasActiveSession()) {
+            $this->app->make('session');
+        }
+
+        return $this->app->make(ResponseFactoryInterface::class)->json(true);
```

Comment:
Can you add a test for this?

Reply:
What should be tested?

Reply:
A short test that makes sure `$this->app->make('session')` is called when there is an active session and not when there isn't. That way we don't accidentally break it in the future since it's not really obvious what's happening here